lib/forms.js

```diff
@@ -13,8 +15,63 @@ var dataTargetImpl = require('./impl/dataTargets');
 var config = require('./config');
 var logger = logfns.getLogger();
 
+function getDbName(uri) {
+  var parsedMongooseUri = mongoUriParser.parse(uri);
+  var dbName = parsedMongooseUri.database;
+  return dbName;
+}
+
+/**
+ * Get a new mongoose connection instance. Will reuse an existing mongoose connection if it's passed in (this connection should be authenticated already and have access to the target db. E.g. admin user). 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)